source/Core/Threads/POWThread.cpp

```diff
@@ -14,10 +15,12 @@
 #include "cmsis_os.h"
 #include "configuration.h"
 #include "main.hpp"
+#include "stdbool.h"
 #include "stdlib.h"
 #include "task.h"
 
 // Small worker thread to handle power (PD + QC) related steps
+bool run_once = true;

```

Comment:
Rather than doing this, just move the FS2711::start() call outside the loop so it runs before the wait loop